polls/templates/polls/htmlform.html

```diff
@@ -0,0 +1,36 @@
+{% if not task %}
+<h3>Come back later for more work.</h3>
+{% else %}
+<h3>Task</h3>
+<form method="POST" id="my-form">
+  {% csrf_token %}
+  <input type="hidden" value="{{ task.pk }}" name="taskPk">
+  <input type="hidden" value="{{ batch_id }}" name="batch_id">
+  <input type="hidden" value="{{ task_list }}" name="tasklist">
+  <input type="hidden" value="{{ task_presentation }}" name="taskPresentation">
+  <h4>ID: {{task.pk}}</h4>
+  <audio controls src="{{ reference_url }}" class="mt-3">
+    Your browser does not support the
+    <code>audio</code> element.
+  </audio>
+  <audio controls src="{{ transform_url }}" class="mt-3">
+    Your browser does not support the
+    <code>audio</code> element.
+  </audio>
+
+  <div class="mt-2">
+    <div class="form-check form-check-inline">
+      <input class="form-check-input" type="radio" id="type1" name="annotationOption" value="XXY" checked>
+      <label class="form-check-label" for="type1">XXY</label>
```

Comment:
XXY and XYX should not be displayed to users

Reply:
We should just have two radio boxes that are unlabeled.

Reply:
So, can we show text like "First and Second were same" or  "First and third sound is same"?